### PR TITLE
Replacement PR for #1349 (neuro images ls -l support) 

### PR DESCRIPTION
**Pull request by olegstepanov**
Had to recreate PR because Azure failed to run the checks

### CODE REVIEW

File: tests/cli/test_ftable.py [hunk 1]
```diff
@@ -58,6 +58,10 @@ def test_multiline(self) -> None:
 
 
 class TestTable:
+    def test_empty(self) -> None:
+        result = list(table(()))
+        assert len(result) == 0
```
Comment (serhiy-storchaka) [inline tests/cli/test_ftable.py]:
```suggestion
        assert result == []
```

Comment (olegstepanov) [reply tests/cli/test_ftable.py]: Wait, table returns an Iterator[str], is it correct to check that it's an empty list?

Comment (serhiy-storchaka) [reply tests/cli/test_ftable.py]: But `result` is a list.

File: neuromation/cli/job.py [hunk 2]
```diff
@@ -545,6 +545,7 @@ def job() -> None:
     """
 
     format = await calc_columns(root.client, format)
+
```
Comment (asvetlov) [inline neuromation/cli/job.py]:
```suggestion
```

File: tests/cli/test_ftable.py (hunk 1 above)
Comment (olegstepanov) [reply tests/cli/test_ftable.py]: Oops, sorry, missed that

File: neuromation/cli/formatters/images.py [hunk 3]
```diff
@@ -133,3 +135,20 @@ def commit_finished(self, data: ImageCommitFinished) -> None:
...
+
+
+class ShortImagesFormatter(BaseImagesFormatter):
+    def __call__(self, images: Iterable[RemoteImage]) -> Iterable[str]:
+        return (str(image) for image in images)
+
+
+class LongImagesFormatter(BaseImagesFormatter):
+    def __call__(self, images: Iterable[RemoteImage]) -> Iterable[str]:
+        rows = [[str(image), image.as_docker_url] for image in images]
```
Comment (asvetlov) [inline neuromation/cli/formatters/images.py]: The last neat: please add the table header, like for `neuro ps` command.

Comment (olegstepanov) [reply neuromation/cli/formatters/images.py]: What would be good headers? 'IMAGE URL', 'DOCKER URL'?

Comment (asvetlov) [reply neuromation/cli/formatters/images.py]: I guess `Neuro URL` and `Docker URL` (both capitalized) -- to reflect the fact that the first value should be used with `neuro`, the second is for `docker`.

**Pull request closed without merging**